frontend/test/metabase/scenarios/admin/troubleshooting/tasks.cy.spec.js

```diff
@@ -6,11 +6,11 @@ describe("scenarios > admin > troubleshooting > tasks", () => {
     cy.signInAsAdmin();
   });
 
-  it.skip("pagination should work (metabase#14636)", () => {
+  it("pagination should work (metabase#14636)", () => {
     // Really ugly way to bump up total number of tasks (but more realistic than using fixture, or stubbing)
     // Since this happens async, that number may vary but it should always be greater than 50 [1] and less than 100 [2]
     // Note: each sync generates 6 tasks and we start with 12 tasks already for the testing sample dataset
-    for (let i = 0; i < 10; i++) {
+    for (let i = 0; i < 13; i++) {
```

Comment:
If `i` is looped only to `10`, test is flaky because the minimum of 50 tasks is not always created.

Tested with `13` a good few times, and it seems to do the trick.